jenkins/test-history/gen_json.py

```diff
@@ -145,7 +145,10 @@ def _get_builds(self, job):
         except (ValueError, TypeError):
             pass
         else:
-            return (str(n) for n in xrange(latest_build, 0, -1))
+            # If it looks like a jenkins build number, return all the indices
+            if latest_build < 20160000000000:
```

Comment:
I feel like this is highlighting the fact that we probably need one place to configure the various builds and buckets we care about, rather than the 4+ we have now.